test/Content.test.js

```diff
@@ -22,14 +27,22 @@ const movieData = [{
 }];
 
 describe('should render Content component', () => {
-    const tree = shallow(<Content movies={movieData} records={10} />);
+    const tree = shallow(
+        <Provider store={store}>
+            <Content movies={movieData} records={10} />
+        </Provider>    
```

Comment:
It's not required to add `Provider` wrapping to make tests work properly.
What was the reason to add it to all tests?

Reply:
Please check this article about writing tests in Redux apps: https://redux.js.org/recipes/writing-tests
It should give you nice overview :)